lib/docker.ts

```diff
@@ -51,24 +51,13 @@ class Docker {
     return await dockerPull.pull(registry, repo, tag, opt);
   }
 
-  public async isDockerExperimentalEnabled(): Promise<boolean> {
-    const output = await subProcess.execute("docker", ["system", "info"]);
-    return output.stdout.includes("Experimental: true");
-  }
-
   public async pullCli(targetImage: string, options?: DockerOptions) {
-    let platformOption: string = "";
+    const opts: string[] = ["pull", targetImage];
     if (options?.platform) {
-      if (await this.isDockerExperimentalEnabled()) {
-        platformOption = `--platform=${options.platform}`;
-      } else {
-        throw new Error(
-          '"--platform" is only supported on a Docker daemon with experimental features enabled',
-        );
-      }
+      opts.push(`--platform=${options.platform}`);
```

Comment:
I'm not sure how smart Docker is with adding the option after the image name (it might be ok, I just don't know and want to make sure it works)

Reply:
It does work, tested that locally (honestly, this should work for every sensible CLI imo, but that's another topic 😄). 

Not sure if it makes sense to test that though? And if yes, how?